metrix-integration/src/main/java/com/powsybl/metrix/integration/MetrixChunk.java

```diff
@@ -133,7 +133,10 @@ public List<TimeSeries> after(Path workingDir, ExecutionReport report) throws IO
 
                         if (networkPointFile != null) {
                             try (Stream<Path> paths = Files.list(workingDir)) {
-                                List<Path> files = paths.filter(path -> path.getFileName().toString().matches(network.getId() + "(.*)\\.xiidm")).collect(Collectors.toList());
+                                List<Path> files = paths.filter(path -> {
+                                    String fileName = path.getFileName().toString();
+                                    return fileName.startsWith(network.getId()) && fileName.endsWith("xiidm");
+                                }).collect(Collectors.toList());
```

Comment:
I would have escaped the parenthesis (and other problematic characters) instead
```suggestion
                                List<Path> files = paths.filter(path -> path.getFileName().toString().matches(Pattern.quote(network.getId()) + "(.*)\\.xiidm")).collect(Collectors.toList());
```

Reply:
(a new import of Pattern might be needed)